src/lib/sitemap/tutorials-content-urls.ts

```diff
@@ -0,0 +1,9 @@
+import { generateTutorialMap } from 'pages/api/tutorials-map'
+import { makeSitemapElement } from './helpers'
+
+export async function allTutorialsUrls() {
+	const tutorialPaths = await generateTutorialMap()
```

Comment:
This function generates paths for all default collection tutorial paths, but doesn't include tutorial landing pages, and collection paths, which I think we would want to add.

Reply:
[This PR has](https://github.com/hashicorp/dev-portal/pull/1684/files#r1119427388) some code you could reference for generating the collection paths. We could eventually refactor to share the logic here.

Reply:
Thanks @kendallstrautman! I'll take a look in the morning